library/oneview_server_profile.py

```diff
@@ -257,7 +257,7 @@ class ServerProfileModule(OneViewModule):
         state=dict(choices=['present', 'absent', 'compliant'], default='present'),
         data=dict(type='dict', required=True),
         params=dict(type='dict', required=False),
-        auto_assign_server_hardware=dict(type='bool', default=True)
+        auto_assign_server_hardware=dict(type='bool', default=False)
```

Comment:
In the discussion with Satya, he said that customer will explicitly mention if he wants auto assign feature to be enabled. So to avoid messing up of things, I changed the default value to False.

Reply:
We need to make sure that the customer's code doesn't break with any of the changes we are making.

Reply:
Ok understood. I have reverted the default value to True to avoid breaking of existing customer code.